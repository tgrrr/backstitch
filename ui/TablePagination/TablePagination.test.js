import React from 'react'
import renderer from 'react-test-renderer'
import { TablePagination } from '.'

it('TablePagination: default', () => {
    const component = renderer.create(
        <TablePagination>
            <>
                TablePagination
            </>
        </TablePagination>
    )
  const tree = component.toJSON()
  expect(tree).toMatchSnapshot()
})
