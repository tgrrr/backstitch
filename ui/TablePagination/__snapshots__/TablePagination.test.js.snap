// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TablePagination: default 1`] = `
<div
  className="sc-AxjAm StDqN"
>
  <div
    className="Tablepagination "
    data-testid="Tablepagination"
  >
    <td
      aria-sort={null}
      className="MuiTableCell-root MuiTablePagination-root"
      colSpan={1000}
    >
      <div
        className="MuiToolbar-root MuiToolbar-regular MuiTablePagination-toolbar MuiToolbar-gutters"
      >
        <div
          className="MuiTablePagination-spacer"
        />
        <p
          className="MuiTypography-root MuiTablePagination-caption MuiTypography-body2 MuiTypography-colorInherit"
        >
          Rows per page:
        </p>
        <div
          className="MuiInputBase-root MuiTablePagination-input MuiTablePagination-selectRoot"
          onClick={[Function]}
        >
          <div
            aria-haspopup="listbox"
            className="MuiSelect-root MuiSelect-select MuiTablePagination-select MuiSelect-selectMenu MuiInputBase-input"
            onBlur={[Function]}
            onFocus={[Function]}
            onKeyDown={[Function]}
            onMouseDown={[Function]}
            role="button"
            tabIndex={0}
          >
            <span
              dangerouslySetInnerHTML={
                Object {
                  "__html": "&#8203;",
                }
              }
            />
          </div>
          <input
            onAnimationStart={[Function]}
            type="hidden"
          />
          <svg
            aria-hidden={true}
            className="MuiSvgIcon-root MuiSelect-icon MuiTablePagination-selectIcon"
            focusable="false"
            viewBox="0 0 24 24"
          >
            <path
              d="M7 10l5 5 5-5z"
            />
          </svg>
        </div>
        <p
          className="MuiTypography-root MuiTablePagination-caption MuiTypography-body2 MuiTypography-colorInherit"
        >
          NaN-NaN of undefined
        </p>
        <div
          className="MuiTablePagination-actions"
        >
          <button
            aria-label="Previous page"
            className="MuiButtonBase-root MuiIconButton-root MuiIconButton-colorInherit"
            disabled={false}
            onBlur={[Function]}
            onClick={[Function]}
            onDragLeave={[Function]}
            onFocus={[Function]}
            onKeyDown={[Function]}
            onKeyUp={[Function]}
            onMouseDown={[Function]}
            onMouseLeave={[Function]}
            onMouseUp={[Function]}
            onTouchEnd={[Function]}
            onTouchMove={[Function]}
            onTouchStart={[Function]}
            tabIndex={0}
            title="Previous page"
            type="button"
          >
            <span
              className="MuiIconButton-label"
            >
              <svg
                aria-hidden={true}
                className="MuiSvgIcon-root"
                focusable="false"
                viewBox="0 0 24 24"
              >
                <path
                  d="M15.41 16.09l-4.58-4.59 4.58-4.59L14 5.5l-6 6 6 6z"
                />
              </svg>
            </span>
          </button>
          <button
            aria-label="Next page"
            className="MuiButtonBase-root MuiIconButton-root MuiIconButton-colorInherit"
            disabled={false}
            onBlur={[Function]}
            onClick={[Function]}
            onDragLeave={[Function]}
            onFocus={[Function]}
            onKeyDown={[Function]}
            onKeyUp={[Function]}
            onMouseDown={[Function]}
            onMouseLeave={[Function]}
            onMouseUp={[Function]}
            onTouchEnd={[Function]}
            onTouchMove={[Function]}
            onTouchStart={[Function]}
            tabIndex={0}
            title="Next page"
            type="button"
          >
            <span
              className="MuiIconButton-label"
            >
              <svg
                aria-hidden={true}
                className="MuiSvgIcon-root"
                focusable="false"
                viewBox="0 0 24 24"
              >
                <path
                  d="M8.59 16.34l4.58-4.59-4.58-4.59L10 5.75l6 6-6 6z"
                />
              </svg>
            </span>
          </button>
        </div>
      </div>
    </td>
  </div>
</div>
`;
